Appsilon/rhino
# Add "Why Rhino?" and clean up main readme

### Changes
Closes #179 

### How to test
Build documentation, see if all links work and all text roughly makes sense.
If something needs addressing, please fix it if possible before merging to speed things up (I'm OOO tomorrow).

After merging, ping @ianemoore to let him know this needs his 👀  :)
